vrecord_policy.xml

```diff
@@ -0,0 +1,93 @@
+    <policy type="or" name="Is the container Matroska or Quicktime?">
+        <policy type="and" name="Is it good Matroska?">
+            <policy type="and" name="Is it new Matroska with an UUID?">
+                <rule name="Is it Matroska?" value="Format" tracktype="General" operator="=">Matroska</rule>
+                <rule name="Matroska version 4 or greater?" value="Format_Version" tracktype="General" operator="&gt;=">4</rule>
+                <rule name="Unique ID is present?" value="UniqueID" tracktype="General"/>
+                <rule name="Is the FFV1 Codec ID V_MS/VFW/FOURCC / FFV1?" value="CodecID" tracktype="Video" operator="=">V_MS/VFW/FOURCC / FFV1</rule>
+                <rule name="Is the PCM Codec ID A_PCM/INT/LIT" value="CodecID" tracktype="Audio" operator="=">A_PCM/INT/LIT</rule>
+                <!-- add Endianness and Signedness when added to report -->
+            </policy>
+            <policy type="or" name="Is it NTSC SD or PAL SD?">
+                <description>Quick test for NTSC vs Pal to validate matrix coefficients. Full NTSC vs PAL test below</description>
+                <policy type="and" name="Is it NTSC?">
+                    <rule name="NTSC FrameRate 29.970?" value="FrameRate" tracktype="Video" operator="=">29.970</rule>
+                </policy>
+                <policy type="and" name="Is it PAL?">
+                    <description>The PAL matrix coefficients value is reported differently depending on container. This test validates for Matroska</description>
+                    <rule name="PAL FrameRate 25.000?" value="FrameRate" tracktype="Video" operator="=">25.000</rule>
+                    <rule name="Are the matrix coefficients BT.470 System B, BT.470 System G?" value="matrix_coefficients" tracktype="Video" operator="=">BT.470 System B, BT.470 System G</rule>
+                </policy>
+            </policy>
+            <policy type="and" name="Is it good Quicktime?">
+                <rule name="Is the file extension mov?" value="FileExtension" tracktype="General" operator="=">mov</rule>
+                <rule name="Is the format MPEG-4?" value="Format" tracktype="General" operator="=">MPEG-4</rule>
+                <rule name="Is the format profile QuickTime" value="Format_Profile" tracktype="General" operator="=">QuickTime</rule>
+                <rule name="Is the FFV1 Codec ID FFV1?" value="CodecID" tracktype="Video" operator="=">FFV1</rule>
+                <rule name="Is the PCM Codec ID in24?" value="CodecID" tracktype="Audio" operator="=">in24</rule>
+                <rule name="Is the endianness Little?" value="Format_Settings_Endianness" tracktype="Audio" operator="=">Little</rule>
+                <rule name="Is the signedness signed?" value="Format_Settings_Sign" tracktype="Audio" operator="=">Signed</rule>
+            </policy>
+            <policy type="or" name="Is it NTSC SD or PAL SD?">
```

Comment:
There is another check of this type up in line 14 that looks the same aside from the rec601 check, but perhaps this is intentional? I"m not really familiar with the and/or policies very well.